apl-core/src/main/java/com/apollocurrency/aplwallet/apl/exchange/model/ExchangeContractStatus.java

```diff
@@ -27,6 +31,10 @@ public boolean isStep2(){
         return this.equals(ExchangeContractStatus.STEP_2);
     }
 
+    public boolean isStep3() {
+        return this.equals(ExchangeContractStatus.STEP_3);
```

Comment:
equals is redundant, just use `==`

Reply:
I think it looks more clear, but in any case inner realization of the "equals" method  is ==, so it won't change performance.

Reply:
No, its not clear because all enum references point to the same instance